### ISSUE TITLE
text box control: assistive technologies cannot navigate through text

### ISSUE THREAD

**Issue by Nuno69** [AvaloniaUI/Avalonia#9770]
**Describe the bug**
Screenreaders do not receive information about cursor changes while navigating throughout the textbox with keyboard.
**To Reproduce**
1. Create any layout containing any number of text boxes.
2. Launch your app and any screenreader (Windows Nrrator and Zhengdu Screenreader tested)
3. Navigate to a textbox using the TAB key and try moving arround the text with the arrow keys. Current cursor position will not be announced.

**Expected behavior**
When moving through the textbox, cursor position should be announced. See the run dialog of Windows for examples.
**Screenshots**

**Desktop (please complete the following information):**

- OS: [e.g. Windows, Mac, Linux (State distribution)] Windows 10
- Version [e.g. 0.10.0-rc1 or 0.9.12] 11 Preview 4

**Additional context**
